t/01-wasm/directives/003-shm_directives.t

```diff
@@ -60,7 +60,7 @@ qr/\[emerg\] .*? \[wasm\] invalid shm size "1x"/
         shm_kv my_kv_1 8192;
     }
 --- error_log eval
-qr/\[emerg\] .*? \[wasm\] shm size of 8192 bytes is too small, minimum required is 12288 bytes/
+qr/\[emerg\] .*? \[wasm\] shm size of 8192 bytes is too small, minimum required is (12288|49152) bytes/
```

Comment:
I think we may be better off with just `\d+` here and in the likes from now on... Perhaps we shouldn't hard-code minimum page sizes of hosts which we have no way of knowing for all hosts when making updates to the test suite. For example if we have a test case that hard-codes size "XYZ", and we then make updates to the code that changes what this "XYZ" output is. We have no way of knowing what the new "XYZ" output is for all hosts, unless we try the test on each host with various page sizes... For instance, my local host machine is a Linux desktop with page size `4096`, but how would I know the page size of all the other hosts this test would run onto? Surely GHA is not going to tell me unless I spend extra time extracting the specific logs from this specific test. Hopefully that makes sense, I think we should be more lose.

Reply:
Yes, it does make sense. It's indeed laborious to maintain these tests preserving the actual numbers.

At first I was committed to preserving them as they help documenting the non trivial underlying behaviour asserted in some test cases, e.g. the ones involving eviction;  this [one](https://github.com/Kong/ngx_wasm_module/blob/main/t/03-proxy_wasm/hfuncs/shm/004-kv_eviction.t#L84) being a good example.

But the documentation of the related directives can also be expanded to include the knowledge being removed from the test cases, if we agree on the need to do so.